lib/sycamore/sycamore/tests/integration/query/test_query_opensearch.py

```diff
@@ -74,6 +75,31 @@ def test_single_query(self, setup_index):
         result = query_executor.query(query)
         assert len(result.hits) > 0
 
+    def test_single_query_with_docset_reader(self, setup_index):
+        context = sycamore.init(
+            exec_mode=ExecMode.LOCAL,
+            params={
+                "opensearch": {
+                    "os_client_args": {
+                        "hosts": [{"host": "localhost", "port": 9200}],
+                        "http_compress": True,
+                        "http_auth": ("admin", "admin"),
+                        "use_ssl": True,
+                        "verify_certs": False,
+                        "ssl_assert_hostname": False,
+                        "ssl_show_warn": False,
+                        "timeout": 120,
+                    },
+                    "index_name": self.INDEX,
+                },
+                "default": {"llm": "some unrelated value"},
+            },
+        )
+        result = context.read.opensearch(
+            query={"query": {"match_all": {}}, "size": 1}, reconstruct_document=True
```

Comment:
Where are you testing `query_kwargs`?

Reply:
this isn't testing query_kwargs, I don't see any tests related to opensearch params